wheatley/bot.py

```diff
@@ -285,6 +286,13 @@ def _on_thats_all(self) -> None:
         # We set this to one, because we expect one clear row between the call and rounds
         self._rows_left_before_rounds = 1
 
+    def _on_rounds(self) -> None:
+        """Callback called when a user calls 'Rounds'."""
+        # We set this to one, because we expect one clear row between the call and rounds
+        self._is_ringing_opening_row = True
+        # except for 'stop at rounds' in which case wheatley just stops, this appears to work best
+        self._rows_left_before_rounds = 1
+
```

Comment:
```suggestion
        # Make next row be the starting row
        self._is_ringing_opening_row = True

```

Reply:
Trouble is that you need ``self._rows_left_before_rounds = 1``` I think,
 or it just gets stuck in rounds

Reply:
Unless you call look to

Reply:
I've assume rounds should put you into the same state as look to

Reply:
You shouldn't need to, `self._rows_left_before_rounds ` is how `That's all` indicates to start ringing the closing rounds. Setting it here would be like calling `Rounds` and `That's all` in RingingRoom.

I suspect the side effect you're seeing is `self._is_ringing_rounds = True` then also being set, which then stops it standings and allows `Go` to take effect. [We should fix that instead](https://github.com/kneasle/wheatley/pull/199#pullrequestreview-676881984)

in `def start_next_row(self, is_first_row: bool)`
https://github.com/kneasle/wheatley/blob/90e52b6390ea4c1a0c2f37d6be3bc879ac1eeb33/wheatley/bot.py#L375-L385

Reply:
Oh I see that's the _on_go  that works..
And line 349

Reply:
Yes that all works - make feel like a noob... well spotted